docs/contributor/01-architecture.md

```diff
@@ -30,11 +30,11 @@ To run, Lifecycle Manager uses the following workflow:
 
 Apart from the custom resources, Lifecycle Manager uses also Kyma, Manifest, and Watcher controllers:
```

Comment:
the controller should be easy to find because we have them at a central location and named properly